pl_examples/hpu_examples/simple_mnist/mnist.py

```diff
@@ -0,0 +1,73 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import torch
+from jsonargparse import lazy_instance
+from torch.nn import functional as F
+
+import pytorch_lightning as pl
+from pl_examples.basic_examples.mnist_datamodule import MNISTDataModule
+from pytorch_lightning.plugins import HPUPrecisionPlugin
+from pytorch_lightning.utilities.cli import LightningCLI
+
+
+class LitClassifier(pl.LightningModule):
+    def __init__(self):
+        super().__init__()
+        self.l1 = torch.nn.Linear(28 * 28, 10)
```

Comment:
(Part of suggestion: let's use torchmetrics `Accuracy` here)

```suggestion
        super().__init__()
        self.l1 = torch.nn.Linear(28 * 28, 10)
        self.test_acc = Accuracy()
```

Reply:
Not going with TorchMetrics as mentioned earlier.

Reply:
could you pls link to the comment about metrics discompatibiity?